<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditorD3DClass</class>
 <widget class="QMainWindow" name="MapEditorD3DClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MapEditorD3D</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRender">
    <property name="title">
     <string>Render</string>
    </property>
    <widget class="QMenu" name="menuRenderMode">
     <property name="title">
      <string>Render Mode</string>
     </property>
     <addaction name="actionSolid"/>
     <addaction name="actionWire"/>
    </widget>
    <addaction name="menuRenderMode"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionImportHeightmap"/>
    </widget>
    <addaction name="menuImport"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionLight"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRender"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSolid">
   <property name="text">
    <string>Solid</string>
   </property>
  </action>
  <action name="actionWire">
   <property name="text">
    <string>Wire</string>
   </property>
  </action>
  <action name="actionOepn">
   <property name="text">
    <string>Oepn</string>
   </property>
  </action>
  <action name="actionImportHeightmap">
   <property name="text">
    <string>Heightmap</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mapeditord3d.qrc"/>
 </resources>
 <connections/>
</ui>
